aspnet/publishing/azure-continuous-deployment.rst

```diff
@@ -1,8 +1,246 @@
+=========================================================
 
-|stub-icon| Publishing to an Azure Web App with Continuous Deployment
-=====================================================================
+By `Erik Reitan`_ 
 
-.. include:: /../common/stub-notice.txt
+This tutorial shows you how to create an ASP.NET 5 web app using Visual Studio and deploy it from Visual Studio to Azure App Service using continuous deployment.
 
-.. _issue: https://github.com/aspnet/Docs/issues/104
+.. contents:: In this article:
+  :local:
+  :depth: 1
+
+Prerequisites
+-------------
+Before you start, make sure that you have the followed the getting started steps for :doc:`/getting-started/installing-on-windows`. This tutorial assumes you have already installed the following:
+
+- `Visual Studio Community 2015 <https://www.visualstudio.com/en-us/downloads/download-visual-studio-vs.aspx>`_
+- `ASP.NET 5  <http://go.microsoft.com/fwlink/?LinkId=627627>`_ (runtime and tooling)
+- `Git <http://git-scm.com/downloads>`_ for Windows
+
+.. note:: For additional information about installing ASP.NET 5, including information about installing on other platforms, see :doc:`/getting-started/index`.
+
+Create an ASP.NET 5 web app
+---------------------------
+
+1. Start Visual Studio 2015.
+
+2. From the **File** menu, select **New** > **Project**. 
+
+3. Select the **ASP.NET Web Application** project template. It appears under **Installed** > **Templates** > **Visual C#** > **Web**. Name the project ``SampleWebAppDemo``. Select the **Add to source control** option and click **OK**.
+
+	.. image:: azure-continuous-deployment/_static/01-new-project.png
+
+4. In the **New ASP.NET Project** dialog, select **Empty** under **ASP.NET 5 Preview Templates**, then click **OK**.
+
+	.. image:: azure-continuous-deployment/_static/02-web-site-template.png
+	
+5. From the **Choose Source Control** dialog box, select **Git** as the source control system for the new project.
+
+	.. image:: azure-continuous-deployment/_static/03-source-control.png
+
+Running the web app locally
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+1. Once Visual Studio finishes creating the app, run the app by selecting **Debug** -> **Start Debugging**. As an alternative, you can press **F5**.
+
+	It may take time to initialize Visual Studio and the new app. Once it is complete, the browser will show the running app.				
+
+	.. image:: azure-continuous-deployment/_static/04-browser-runapp.png
+
+2. After reviewing the running Web app, close the browser and click the "Stop Debugging" icon in the toolbar of Visual Studio to stop the app.
+
+.. note:: To review the ASP.NET 5 project, see the "Review the project" section of :doc:`/tutorials/your-first-aspnet-application`.
+
+Create a web app in the Azure Portal
+------------------------------------
+
+The following steps will guide you through creating a web app in the Azure Portal.
+
+1. Log in to the `Azure Portal <https://portal.azure.com>`_.
```

Comment:
What if the reader doesn't have an Azure account? Can we point them to create a free account?